src/app/page.tsx

```diff
@@ -1,19 +1,39 @@
 //db imports
-import { db, User, NewUser } from './db/dbstuff';
-import { users } from './db/schema';
+import { db, Race } from './db/dbstuff';
+import { races } from './db/schema';
 
 //link import
 import Link from 'next/link'
 
-const result: User[] = await db.select().from(users);
+import './homestyles.css'
+
+const result: Race[] = await db.select().from(races);
 
 export default function Home() {
-  const output = result.map((runner,index) => <div>{runner.username} & {runner.firstname} & {runner.lastname}</div>);
+  //build a list of races based on the race table, then pass the race object as a parameter to the component
+  const racelist = result.map((race,index) => {return(<RaceComponent myrace={race} key={index}/>);});
   return (
     <div>
       <h1>Landing/Race List</h1>
-      <li><Link href="/races">Example Race {"(Link won't work due to eventual dynamic routing)"}</Link></li>
-      {output}
+      {racelist}
     </div>
   )
 }
+
+//pull a couple fields out of the object to populate the list item
+function RaceComponent(props: any){
+  const race: Race = props.myrace;
+  const start = race.starttime;
+  const fullDate = `${start?.getMonth()}/${start?.getDate()}/${start?.getFullYear()}`;
+  const fullTime = `${start?.getHours()}:${start?.getMinutes()}`;
+
+  //return our data from the db plus a dynamic route link based on the race ID
```

Comment:
add comment for next person looking at this:
// see db/schema.ts for which fields you have access to!